hw7/tests/task72_test.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from hw7.tasks.task72 import yeild_str, backspace_compare
+from typing import List
+
+
+@pytest.mark.parametrize(
+    ["input_str", "expected_result"],
+    [
+        ("12345", ["5", "4", "3", "2", "1"]),
+        ("#1###2#3#4##5", ["5"]),
+        ("", []),
+        ("###", []),
+    ],
+)
+def test_yeild_str(input_str: str, expected_result: List[str]):
+    actual_result = [item for item in yeild_str(input_str)]
+    assert actual_result == expected_result
+
+
+def test_positive_backspace_compare_with_example_value_1():
+    assert backspace_compare(s="ab#c", t="ad#c") == True
```

Comment:
it's enough to check "assert x" and "assert not x".
"== True" and "== False" are the same

Reply:
Did it in that way, because Andrey said, that tests should be as clearer as possible